ReactiveCocoaTests/Swift/SignalSpec.swift

```diff
@@ -36,71 +36,6 @@ class SignalSpec: QuickSpec {
 				expect(signal).to(beNil())
 			}
 
-			it("should deallocate after erroring") {
-				weak var signal: Signal<AnyObject, TestError>? = Signal { observer in
-					testScheduler.schedule {
-						observer.sendFailed(TestError.Default)
-					}
-					return nil
-				}
-				
-				var errored = false
-				
-				signal?.observeFailed { _ in errored = true }
-				
-				expect(errored).to(beFalsy())
-				expect(signal).toNot(beNil())
-				
-				testScheduler.run()
-				
-				expect(errored).to(beTruthy())
-				expect(signal).to(beNil())
-			}
-
-			it("should deallocate after completing") {
```

Comment:
@vadim-fueled Good idea. It works. Glad that Xcode 7 introduced `@testable` which grants access to `internal` stuff. :)